File: src/app/core/types/banner.ts
export class Banner {
  id: number;
  title: string;
  description: string;
  image: string;
  imageUrl: string;
  imageCrop: string;
  imageCropUrl: string;
  imageExtension: string;

  constructor(id: number, title: string, description: string, image: string,
              imageUrl: string, imageCrop: string, imageCropUrl: string, imageExtension: string) {
    this.id = id;
    this.description = description;
    this.title = title;
    this.image = image;
    this.imageUrl = imageUrl;
    this.imageCrop = imageCrop;
    this.imageCropUrl = imageCropUrl;
    this.imageExtension = imageExtension;
  }
}

File: backend/Marbomil.Infrastructure/Entities/Database/Banner.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Marbomil.Infrastructure.Entities.Database
{
    public class Banner : IDbEntity
    {
        public int Id { get; set; }

        [MaxLength(100)]
        public string Title { get; set; }

        public string Description { get; set; }

        [MaxLength(1000)]
        public string ImageUrl { get; set; }

        [MaxLength(1000)]
        public string ImageCropUrl { get; set; }
    }
}
